src/main.c

```diff
@@ -94,6 +95,9 @@ void main(int argc, char **argv)
       {
       }
     }
+	
+    Pdu_Manager_Handle_Rx();
```

Comment:
I would prefer to slot this in over in per_task.c:Task_Ready()
The idea is to keep main.c at a minimum because it tends to aggregate platform/compiler dependencies.